__init__.py

```diff
@@ -267,23 +280,45 @@ def Model_Event(position):
 
 
 class BrowseToModelOperator(Operator):

```

Comment:
Small thing: We should probably rename this class since it imports now instead of opening the model